doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
+
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type Date, the resulting DMN field is not of a DMN date type.
 
 Steps to reproduce:
 
-* Create two tests where the second test is created according to the hint message in the first test, while retaining the same input as the first test.
+. Open a project in {CENTRAL}.
+. Add a Java data object.
+. Create a `java.util.Date` field in this object.
+. Save and close the Java class.
+. Open a DMN file.
+. Select the *Data Types* tab.
+. Click *Import Data Object*.
+. Select the Java class that you created.
+. Click *Import*.
+. Check the type of the field into which the `java.util.Date` field was converted.
 
-Expected result: The corrected output has the correct format, as shown in the following example:
+Expected result: The field has the type `date` or `date and time`.
 
-`{"key_a" : "value_a", "key_b" : "value_b", ...}`
+Actual result: The field has the type `Any`.
 
-Actual result: The corrected output has an incorrect format (missing double quotes), as shown in the following example:
+Workaround: Manually change the type of the field to `date` or `date and time`.
 
-`{key_a : value_a, key_b : value_b, ...}`
+.When importing a data object from a Java class, a field of the Java List type is not converted to the DMN collection type [https://issues.redhat.com/browse/RHDM-1144[RHDM-1144]]
 
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type List, the resulting DMN field is not of a DMN collection type.
 
+Steps to reproduce:
 
-== Decision engine
+. Open a project in {CENTRAL}.
+. Add a Java data object.
+. Create a `List` field in this object.
+. Save and close the Java class.
+. Open a DMN file.
+. Select the *Data Types* tab.
+. Click *Import Data Object*.
+. Select the Java class that you created.
+. Click *Import*.
+. Check the type of the field into which the `List` field was converted.
 
-.The executable model does not fully parse multi-line patterns [https://issues.jboss.org/browse/RHDM-1098[RHDM-1098]]
+Expected result: The field has the type `collection`.
 
-Issue: When a pattern has multiple lines, the executable model does not fully parse its conditions. Only the first line is parsed.
+Actual result: The field has the type `Any`.
 
-NOTE: This issue is resolved with {PRODUCT} patch release 7.5.1.
+Workaround: Manually change the type of the field to `collection`.
 
-Workaround: None.
 
-endif::[]
+.When you change a decision table header in the properties panel, the change is not saved [https://issues.redhat.com/browse/RHDM-1181[RHDM-1181]]
 
-== High availability
+Issue: In the DMN designer, when editing a decision table, you can click a header to view its properties panel. However, if you change settings for the header in the properties panel, the change is not saved.
```

Comment:
In the DMN designer, when you edit a decision table you can click...
when editing: dangling modifier